packages/sui-theme/src/utils/_spacing.scss

```diff
@@ -0,0 +1,74 @@
+// DESCRIPTION:
+//
+// Provides a basic spacing classes library in emmet convection
+//
+// USAGE:
+//
+// In your vertical theme...
+//
+// @each $side in $m-sides {
+//   @include create-space('margin', $side, $p-sizes);
+// }
```

Comment:
Maybe we can include this in a file, so the marketplace will only need to import a file. Plus, we can modify it in the future without needing to modify the marketplaces code.